Cache.podspec

```diff
@@ -1,7 +1,7 @@
 Pod::Spec.new do |s|
   s.name             = "Cache"
   s.summary          = "Nothing but cache."
-  s.version          = "4.1.2"
+  s.version          = "4.2.0"
```

Comment:
Why a new version? Are there any code changes?

Reply:
@vadymmarkov It's to make sure there's no potential conflicts